api/python/provisioner/inputs.py

```diff
@@ -467,54 +467,61 @@ class Release(ParamGroupInputBase):
-    primary_ip: str = ParamGroupInputBase._attr_ib(
-        _param_group, descr=" Controller A IP",
+    controller_primary_mc_ip: str = ParamGroupInputBase._attr_ib(
+        _param_group, descr="Controller A IP",
         validator=Validation.check_ip4
     )
-    secondary_ip: str = ParamGroupInputBase._attr_ib(
-        _param_group, descr=" Controller B IP",
+    controller_secondary_mc_ip: str = ParamGroupInputBase._attr_ib(
+        _param_group, descr="Controller B IP",
         validator=Validation.check_ip4
     )
     controller_user: str = ParamGroupInputBase._attr_ib(
-        _param_group, descr=" Controller user"
+        _param_group, descr="Controller user"
     )
     # TODO IMPROVE EOS-14361 mask secret
     controller_secret: str = ParamGroupInputBase._attr_ib(
-        _param_group, descr=" Controller password"
+        _param_group, descr="Controller password"
     )
     controller_type: str = ParamGroupInputBase._attr_ib(
-        _param_group, descr=" Controller type"
+        _param_group, descr="Controller type"
+    )
+    storage_metadata_device: str = ParamGroupInputBase._attr_ib(
+        _param_group, descr="Storage Metadata Device"
+    )
+    storage_data_devices: str = ParamGroupInputBase._attr_ib(
+        _param_group, descr="Storage Data Devices"
+    )
+    hostname: str = ParamGroupInputBase._attr_ib(
+        _param_group, descr="Node Hostname"
     )
 
 
 @attr.s(auto_attribs=True)
 class StorageEnclosure(ParamGroupInputBase):
-    controller_a_ip: str = StorageEnclosureParams.primary_ip
-    controller_b_ip: str = StorageEnclosureParams.secondary_ip
+    type: str = StorageEnclosureParams.type
+    controller_a_ip: str = StorageEnclosureParams.controller_primary_mc_ip
+    controller_b_ip: str = StorageEnclosureParams.controller_secondary_mc_ip
     controller_user: str = StorageEnclosureParams.controller_user
     controller_secret: str = StorageEnclosureParams.controller_secret
+    storage_metadata_device: str = StorageEnclosureParams.storage_metadata_device
+    storage_data_devices: str = StorageEnclosureParams.storage_data_devices
+    hostname: str = StorageEnclosureParams.hostname
 
 
 class NodeNetworkParams():
     _param_group = 'node'
-    hostname: str = ParamGroupInputBase._attr_ib(
-        _param_group, descr="node hostname"
-    )
-    roles: List = ParamGroupInputBase._attr_ib(
-        _param_group, descr="List of roles assigned to the node"
+    type: str = ParamGroupInputBase._attr_ib(
+        _param_group, descr="Common Server configuration"
     )
-    data_public_interfaces: List = ParamGroupInputBase._attr_ib(
-        _param_group, descr="node data public network interfaces"
+    cluster_id: str = ParamGroupInputBase._attr_ib(
+        _param_group, descr="Cluster ID"
     )
-    data_private_interfaces: List = ParamGroupInputBase._attr_ib(
-        _param_group, descr="node data private network interfaces"
+    network_data_interfaces: List = ParamGroupInputBase._attr_ib(
```

Comment:
We need the demarcation for private and public interfaces.
Moreover, do we really need to prepend network here?
I believe it's handled in `configure_setup.py`.

Avoid major changes.

Reply:
Done. Kept the naming convention as such, and only made few changes for the new `config.ini` format.